app/serializers/basic_post_serializer.rb

```diff
@@ -44,8 +44,12 @@ def cooked
   end
 
   def ignored
-    object.is_first_post? && IgnoredUser.where(user_id: scope.current_user&.id,
-                                               ignored_user_id: object.user_id).exists?
+    object.is_first_post? &&
+      scope.current_user&.id != object.user_id &&
+      !object.user.admin? &&
```

Comment:
You can use `user.staff?` to handle `admin` and `moderator` cases at the same time.